strongbox-web-core/src/test/java/org/carlspring/strongbox/forms/configuration/RepositoryFormTestIT.java

```diff
@@ -362,4 +365,19 @@ void testRemoteRepositoryFormInvalidNegativeHttpConnectionPool()
 
         validateAndAssert(repositoryForm, 1, "A httpConnectionPool must be positive or zero.");
     }
+
+    private String getPatternLocalisedMessage(String expectedPattern)
+    {
+        String message;
+        try {
```

Comment:
Cosmetic: Braces on new lines please. :)

For more details, please check our [coding convention](https://strongbox.github.io/developer-guide/coding-convention.html). Thanks! :)

Reply:
Formatted